src/osd/ReplicatedBackend.cc

```diff
@@ -1010,6 +1010,10 @@ void ReplicatedBackend::do_repop(OpRequestRef op)
 	   << " " << m->logbl.length()
 	   << dendl;
 
+  if (cct->_conf.get_val<bool>("osd_debug_repop_crash")) {
```

Comment:
the get_val<> method does a lookup in a std::map, so it's slow and can't be used in any hot path.  Instead, add osd_debug_repop_crash to common/legacy_options.h and use ``cct->_conf->osd_debug_repop_crash``

Reply:
thanks for teaching!